pkg/setup/setup.go

```diff
@@ -114,7 +113,7 @@ func Run(ctx context.Context, config ServerConfig) {
 	pv, handler, _ := metrics.Init()
 	ctx = metrics.WithProvider(ctx, pv)
 
-	pv.Meter("setup").Int64ObservableGauge("background_job_ready", metric.WithInt64Callback(func(_ context.Context, o metric.Int64Observer) error {
```

Comment:
What is the lint actually complaining about?

Reply:
Return value not checked. I couldn't figure out exactly how to do so.